cv_next/providers/supabase-provider.tsx

```diff
@@ -32,7 +32,7 @@ export default function SupabaseProvider({
       data: { subscription },
     } = supabase.auth.onAuthStateChange((_, _session) => {
       if (_session?.access_token !== session?.access_token) {
-        router.refresh()
+        // router.refresh()
```

Comment:
Is this intentional? Should this be merged like that?

Reply:
This entire file as well as other files that are related to the supabase auth, are getting removed in with the SSR branch